src/I18n/I18n.php

```diff
@@ -206,22 +206,45 @@ public static function config($name, callable $loader)
      * When called with no arguments it will return the currently configure
      * locale as stored in the `intl.default_locale` PHP setting.
      *
+     * @deprecated Deprecated since 3.5. Use setLocale() and getLocale().
```

Comment:
You dont have to repeat the deprecated part :)
The consistent way across the newer deprecations so far always was:

    @deprecated 3.4.0 Use setTypes()/getTypes() instead.